biowdl_lint.sh

```diff
@@ -1,12 +1,22 @@
 #!/usr/bin/env bash
 
 # Validate every WDL file in the repository with womtool validate and miniwdl check.
+# Also check parameter_meta presence unless the argument "skip-wdl-aid" is given.
 set -e
 for WDL_FILE in $(git ls-files *.wdl)
   do
     echo $WDL_FILE
     womtool validate $WDL_FILE
     miniwdl check $WDL_FILE
+    if [[ "$1" != "skip-wdl-aid" ]]
```

Comment:
Isn't it easier to set this with an environment variable? 
That way we can control it without modifying the script call.

Reply:
You mean replace the `$1` with an environment variable? I'm not sure how to set those in Jenkins :stuck_out_tongue:

Reply:
On second thought, I think this `if` can just be removed. It was there because the tasks repo failed since there are no workflows in those wdl files. But since there are some wdl files without workflows in other repos as well (structs.wdl), I added the grep logic to ignore the no-workflow-errors. That should mean tasks will succeed now as well.